bench_test/CMakeLists.txt

```diff
@@ -31,9 +37,6 @@ macro(define_runtime_algebra_test name)
     cm_test(NAME ${test_name} SOURCES ${name}.cpp ARGS ${additional_args})
 
     target_include_directories(${test_name} PRIVATE
-                               "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
-                               "$<BUILD_INTERFACE:${CMAKE_BINARY_DIR}/include>"
-
                                ${Boost_INCLUDE_DIRS})
```

Comment:
No need in this inlcudes. They are already declared for tested target:
https://github.com/NilFoundation/crypto3-algebra/blob/087efb2c2cdb89e3525bd999b7ceba2e9f9a85ce/CMakeLists.txt#L67-L71